string-manipulation-2/src/capitalize-word.js

```diff
@@ -0,0 +1,11 @@
+/* exported capitalizeWord */
+function capitalizeWord(word) {
+  var smallWord = word.toLowerCase();
+  if (smallWord === 'javascript') {
+    return 'JavaScript';
+  }
+  var lastIndex = smallWord.length;
+  var firstChar = smallWord[0].toUpperCase();
+  var remaining = smallWord.slice(1, lastIndex);
```

Comment:
If you just pass a start point and no end point, `slice` will take the remainder of the string automatically. You can revise this to just `var remaining = smallWord.slice(1);`